.golangci.yml

```diff
@@ -0,0 +1,28 @@
+run:
+  skip-dirs:
+    - vendor
+
+  skip-files:
+    - ".*\\_test.go$"
+
+linters:
+  # refer https://golangci-lint.run/usage/linters/
+  disable-all: true
```

Comment:
Shouldn't be better to enable all the linters and detect the linters which are not passing?

Something like:
```yaml
  enable-all: true
  disable:
    - interfacer  # Deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner.
    - golint  # Deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive.
    - scopelint  # Deprecated (since v1.39.0) due to: The repository of the linter has been deprecated by the owner.  Replaced by exportloopref.
    - maligned  # Deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner.  Replaced by govet 'fieldalignment'.
    - exhaustivestruct  # Deprecated (since v1.46.0) due to: The owner seems to have abandoned the linter.  Replaced by exhaustruct.
    - nosnakecase  # Deprecated (since v1.48.1) due to: The repository of the linter has been deprecated by the owner.  Replaced by revive(var-naming).
    - ifshort  # Deprecated (since v1.48.0) due to: The repository of the linter has been deprecated by the owner.
    - deadcode  # Deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
    - varcheck  # Deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
    - structcheck  # Deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
```

Reply:
Good alternate suggestion, From a readability perspective and to get familiar initially, it is better to call out the list of linters enabled until community comes upto speed.